documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -545,10 +545,10 @@ https://www.postgresql.org/docs/current/static/sql-altertable.html#SQL-CREATETAB
 
 There are 4 possible values for `REPLICA IDENTITY`:
 
-* DEFAULT - `UPDATE` and `DELETE` events will only contain the previous values for the primary key columns of a table
+* DEFAULT - `UPDATE` and `DELETE` events will only contain the previous values for the primary key columns of a table, in case of `UPDATE` only the primary columns with changed values are present
```

Comment:
So for a DELETE all PK columns will be in the `before` block?

Reply:
@gunnarmorling Yes, I've tried it